bindings/python/capstone/__init__.py

```diff
@@ -174,7 +182,9 @@
 CS_ARCH_WASM = 13
 CS_ARCH_BPF = 14
 CS_ARCH_RISCV = 15
-CS_ARCH_MAX = 16
+# CS_ARCH_SH = 16
```

Comment:
Because `CS_ARCH_TRICORE=17` corresponds to C code. Then there is a missing `CS_ARCH_SH=16` in between, but actually `CS_ARCH_SH` does not have Python bindings.

Reply:
> > please put all the header code under include/capstone/inc/ into one file, just like the other archs.
> 
> @aquynh Could you point out what the advantage of this is? It kinda breaks the idea of `auto-sync` that generated content and hand written content is clearly separated into unique files.

i prefer to have it consistent with other archs.

is it too hard to update your tool to put all of the inc files into one?

Reply:
> since all LLVM based architectures (with the exception of x86) will be updated to use auto-sync, they are consistent to each other.

I agree with this.

Reply:
I have an idea, maybe we can do it like this:

we insert comments like this in `<arch>.h`:

```
// generate content <section> begin
...
// generate content <section> end
```

tablgen replaces these comments with the actual content when running.

Reply:
Something like this would be the way I'd implement this.
But implementing is not so much a problem.
It is the decision what is more important.

- A single header file for each architecture.

Or

- Following the auto-sync rules for file updating.

Reply:
I personally think we should switch to the new scheme, especially since there are plans to update other architectures, and add new ones (e.g. Xtensa, LANAI). The more architectures capstone has, the harder it will become to combine autogenerated and manual content in the same file.

Reply:
@aquynh so what is your decision?

Reply:
@XVilka Well, he insist:)